src/render/tutorials/getting-started-with-docker/2_0.html.md

```diff
@@ -33,120 +33,151 @@ Project has the following structure:
-
-### HTML Template
-
-Create a `knotx/repository/content` directory in your project and put there following page template with Knot.x snippet (`<knotx:snippet data-knotx-task="bookslist">...`):
-
-*books.html*
-```html
-<!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Knot.x</title>
-  <link href="https://bootswatch.com/4/superhero/bootstrap.min.css" rel="stylesheet"/>
-</head>
-<body>
-<div class="container-fluid">
-  <div class="row">
-    <div class="col-sm-12">
-      <div class="panel panel-default">
-        <div class="panel-heading">Books list</div>
-        <div class="panel-body">
-          This page lists Java related books provided by <a
-            href="https://www.googleapis.com/books/v1/volumes?q=java">Googleapis book service</a>.
-        </div>
-      </div>
-    </div>
-  </div>
-  <div class="row">
-   <knotx:snippet data-knotx-task="bookslist">
-      {{#each books._result.items}}
-        <div class="col-sm-3">
-        <div class="card">
-          <img class="card-img-top"
-             src="{{volumeInfo.imageLinks.thumbnail}}"
-             alt="{{volumeInfo.title}}">
-          <div class="card-block">
-            <h4 class="card-title">{{volumeInfo.title}}</h4>
-            <p class="card-text">
-              {{#each volumeInfo.authors as |author|}}
-                {{author}}{{#unless @last}}, {{/unless}}
-              {{/each}}<br />
-              Published: {{volumeInfo.publishedDate}}
-            </p>
-          </div>
-         </div>
-      </div>
-      {{/each}}
-    </knotx:snippet>
-
-    </div>
-</div>
-</body>
-</html>
-```
-
-### Templates repository configuration
-We will use [fsRepoConnectorHandler](https://github.com/Knotx/knotx-repository-connector/tree/master/fs), its purposes are purely academical.
-It is not designed to be used as a production ready solution (you should use [httpRepoConnectorHandler](https://github.com/Knotx/knotx-repository-connector/tree/master/http) there).
-
-Create `knotx/conf/routes/handlers/fsRepoConnectorHandler.conf` file with the following:
-```hocon
-# Path to the directory on the local filesystem which will be the root for requested templates
-catalogue = "/usr/local/knotx/repository/"
-```
-This way we define the `repository` directory that will be our files content repository.
+## Create a new routing path
 
-### openapi.yml
-Open `knotx/conf/openapi.yml` and add following path definition:
+Let's take a look at `knotx/conf/openapi.yaml` file. 
 
-```
-  /content/*:
+This is a HOCON (*Human-Optimized Config Object Notation*) file. More on HOCON  can be found in [this informal specification](https://github.com/lightbend/config/blob/master/HOCON.md).
```

Comment:
Actually this is a YAML file. `operations.conf` is a HOCON file

Reply:
whoops! 🙉